ibis/omniscidb/ddl.py

```diff
@@ -340,6 +341,83 @@ def compile(self):
         return self._wrap_command(action)
 
 
+class AddColumn(AlterTable):
+    """Add Column class."""
+
+    def __init__(self, table_name, **columns_with_types):
+        super().__init__(table_name)
+        self.dict_cols_with_types = {}
+        for key, value in columns_with_types.items():
+            self.dict_cols_with_types[key] = value
```

Comment:
not sure if I am following you here but maybe you can replace that to:

`self.dict_cols_with_types = columns_with_types`

Reply:
I replaced argument to the dictionary and added line proposed by you